lib/site_prism/element_container.rb

```diff
@@ -23,6 +23,10 @@ def elements(collection_name, *find_args)
     end
     alias collection elements
 
+    def expected_elements(*elements)
+      @expected_items = elements.map(&:to_s)
```

Comment:
If these aren't mapped to strings and remain as symbols does anything happen?

Reply:
@TheMetalCode before I merge in can you just answer this question

Reply:
@luke-hill Sorry, missed this earlier. The reason we want `to_s` here is that they end up getting evaluated in the `all_there?` check like so: `elements_to_check.all? { |element| send "has_#{element}?" }`. So if `element` wasn't already a string at that point, that `send` call wouldn't work.

Reply:
For reference they "should". Interpolating symbols gets converted into strings and even running send on symbols should work. 

But hey I've merged this so it doesn't matter now.